core/src/main/antlr4/com/databricks/labs/remorph/parsers/tsql/TSqlParser.g4

```diff
@@ -1860,6 +1860,7 @@ createTable
     : CREATE TABLE tableName LPAREN columnDefTableConstraints (COMMA? tableIndices)* COMMA? RPAREN (
         LOCK simpleId
     )? tableOptions* (ON id | DEFAULT | onPartitionOrFilegroup)? (TEXTIMAGE_ON id | DEFAULT)? SEMI?
+    | CREATE TABLE tableName (LPAREN columnNameList RPAREN)? tableOptions? AS selectStatementStandalone SEMI?
```

Comment:
create two rules - `createTableDefault` and `createTableAs`. this will simplify the parser. add unit and functional tests